tests/testqml.cpp

```diff
@@ -1,47 +1,29 @@
 // SPDX-License-Identifier: GPL-2.0
-#include <QApplication>
-#include <QQmlContext>
-#include <QQmlEngine>
-#include <QtQuickTest>
-#include <QtTest>
-
-#include "core/qt-gui.h"
+#include "testqml.h"
 #include "core/settings/qPref.h"
 
-// this is the content of QUICK_TEST_MAIN amended with
-// registration of ssrf classes
+#include <QtQuickTest>
+
+// main loosely copied from QUICK_TEST_MAIN_WITH_SETUP macro
 int main(int argc, char **argv)
 {
-	// QML testing is not supported in the oldest Qt versions we support
-	// if running with Qt version less than 5.10 then skip test
-#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
+#if QT_VERSION < QT_VERSION_CHECK(5, 11, 0)
+	return 0;
+#else
```

Comment:
so now this requires 5.11 ? 
That means it won't run on Travis at all.

Reply:
Sorry but using contextProperty requires access to the QML engine, and this is not supported in earlier versions.

If we really need to support earlier Qt versions for mobile, then I can copy/modify the Qt sources to do the same...whatever you prefer.

Reply:
I'll work on making sure that our mobile versions are built with Qt 5.11
Not sure how we'll get this into Travis, but I guess I'll work on that as well.

Reply:
I'm trying hard to have Travis run something really similar to what I use to build the app with on iOS.
While I don't want to lose the old Qt versions, for Android and iOS I am much more willing to push for "latest and greatest". The big challenge was that when 5.11 came out packaging the app on both platforms failed. I am once again trying to get this to work, now that I got some hints from another app  Qt/QML app packager. We'll see how this goes (but Tue-Thu I am traveling with reduced time for Subsurface)